tests/conftest.py

```diff
@@ -10,29 +10,40 @@
 
 GIT_ROOT = os.path.join(os.path.abspath(os.path.dirname(__file__)), '..')
 
-
 class HlwmBridge:
 
     HC_PATH = os.path.join(GIT_ROOT, 'herbstclient')
 
-    def __init__(self,display):
+    def __init__(self, display, hlwm_process):
         self.next_client_id = 0;
         self.env = {
             'DISPLAY': display,
         }
+        self.hlwm_process = hlwm_process
+        self.hc_idle = subprocess.Popen(\
+                    [self.HC_PATH, '--idle', 'rule', 'here_is_.*'],
+                    bufsize=1, # line buffered
```

Comment:
Did you confirm that this really enables line-buffering? The docs make it sound like it requires `universal_newlines=True` to work.

Reply:
On the other hand, line buffering is probably the default anyway?

Reply:
I agree with both of your comments and the documentation https://docs.python.org/3/library/subprocess.html really sounds a bit contradictory. So maybe lets set `universal_newlines=True` to be on the save side?

Reply:
Seems reasonable :+1: